<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OnTheSpot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../../../../../../../../resources/icon.png</normaloff>../../../../../../../../../../../../../../resources/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox #group_login_form_3, #gb0_0, #gb0_1, #gb0_2, #gb0_3, #gb0_4, #gb0_5, #gb0_6, #gb0_7, #gb1_0, #gb1_1, #gb1_2, #gb1_3, #gb1_4, #gb1_5, #gb1_6, #gb1_7, #gb1_8, #gb1_9, #gb1_10, #gb1_11, #gb1_12, #gb1_13, #gb1_14, #gb1_29, #gb2_0, #gb2_1, #gb2_2, #gb2_3, #gb2_4, #gb2_5, #gb2_6, #gb2_7, #gb2_8, #gb2_9, #gb2_10, #gb2_11, #gb2_12, #gb2_13, #gb2_14, #gb2_15, #gb2_16, #gb2_17, #gb2_18, #gb3_0, #gb3_1, #gb3_2, #gb3_3, #gb3_4, #gb3_5, #gb3_6 { border: 0;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabview">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="loadcsv">
       <attribute name="title">
        <string>Load CSV</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="group_search">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border:0;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="btn_load_csv">
                <property name="text">
                 <string>Load CSV (Follow Format Below)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_temp_dl_root">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
	border:0;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_28">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_search_results">
          <property name="title">
           <string>Results</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Phone Number (999-999-9999)</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sendsms">
       <attribute name="title">
        <string>Send SMS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_pending_phone_number">
          <property name="text">
           <string>To: {phone_number}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="label_pending_message">
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_29">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_previous_entry">
             <property name="text">
              <string>Previous Entry</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_next_entry">
             <property name="text">
              <string>Next Entry</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_send_sms">
             <property name="styleSheet">
              <string notr="true">background-color: #00FF00</string>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QScrollArea" name="settings_scroll_area">
          <property name="styleSheet">
           <string notr="true">QGroupBox #scrollArea, #adv_settings_scrollarea {border: 0;}</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="adv_settings_scrollarea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>670</width>
             <height>410</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QGroupBox" name="gb0">
              <property name="styleSheet">
               <string notr="true">QGroupBox { font-weight: bold; }</string>
              </property>
              <property name="title">
               <string>General</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QGroupBox" name="gb1_6">
                 <property name="title">
                  <string/>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_35">
                  <item>
                   <widget class="QGroupBox" name="group_lyrics_cfg_only_synced_7">
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_53">
                     <item>
                      <widget class="QLabel" name="lb_illegal_character_replacement_6">
                       <property name="text">
                        <string>Your Telus Phone Number</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="label_settings_personal_telus_number">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="placeholderText">
                        <string>Must be formatted as 416-234-5678</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb1_4">
                 <property name="title">
                  <string/>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_32">
                  <item>
                   <widget class="QGroupBox" name="group_lyrics_cfg_only_synced_3">
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_48">
                     <item>
                      <widget class="QLabel" name="lb_illegal_character_replacement">
                       <property name="text">
                        <string>Telus Token</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="label_settings_token"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Available Variables: {name}, {phone_number}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="gb1_10">
                 <property name="title">
                  <string/>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_39">
                  <item>
                   <widget class="QGroupBox" name="group_lyrics_cfg_only_synced_6">
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_50">
                     <item>
                      <widget class="QLabel" name="lb_illegal_character_replacement_2">
                       <property name="text">
                        <string>Message</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPlainTextEdit" name="label_settings_message">
                       <property name="plainText">
                        <string>Hi {name}, I was just wondering if you were interested in commercial truck warranty?</string>
                       </property>
                       <property name="placeholderText">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toggle_theme_button">
          <property name="text">
           <string>Set Theme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_24">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_reset_config">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 0, 0, 100);</string>
             </property>
             <property name="text">
              <string> Reset Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_save_config">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string> Save Settings</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
